log/log_test.go

```diff
@@ -90,6 +90,8 @@ func TestHuman(t *testing.T) {
 		t.Error(err)
 	}
 
+	l.WrapError(errors.New("hello error")).Log("msg", "there is a big error")
+
```

Comment:
What does `TestHuman` actually test?

Reply:
test grammer~ and give you a easy way to see what output looks like...